### DISCUSSION TITLE
Second instance. Quitting.

### DISCUSSION THREAD

**Discussion by tomasmorys** [swar/Swar-Chia-Plot-Manager#407] [Q&A]
Hi, my first run will not start and I have in the log: ' Second instance. Quitting. ' Thanks for the help.

**Comment by boemxp** [comment]
I just figure it's out that the log show 'Second instance. Quitting.' because setting up wrong "chia_location: "

To reproduce the case:
in config.yaml
chia_location: C:\Users\<USER>\AppData\Local\chia-blockchain\chia.exe

Log >> Second instance. Quitting

Solution
Setting the corected "chia location"
chia_location: C:\Users\<USER>\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe

Chia Plot Manager can run as normal

**Reply by tomasmorys** [reply]
Thanks !

**Reply by descovi** [reply]
This is the correct solution, same error here :)
